packages/js/src/core/user-info.ts

```diff
@@ -0,0 +1,14 @@
+import { createRequester, Requester } from '../utils/requester';
+
+export type UserInfoResponse = {
+  sub: string;
```

Comment:
only sub being returned?

Reply:
yes, only sub for now
https://www.notion.so/silverhand/oidc-me-3a281ed217894558be0651e983f9ac2c